demos/projects/ESPRESSIF/aziotkit/README.md

```diff
@@ -77,9 +77,36 @@ Parameter | Value
 ---------|----------
  `Azure IoT Hub FQDN` | _{Your Azure IoT Hub Host FQDN}_ (Unused if Device Provisioning is enabled below)
  `Azure IoT Device ID` | _{Your Azure IoT Hub device ID}_
- `Azure IoT Device Symmetric Key` | _{Your Azure IoT Hub device symmetric key}_
  `Azure IoT Module ID` | _{Your Azure IoT Hub Module ID}_ (optional, specify module id if using a device module; else leave blank if not)
 
+Select your desired authentication method with the `Azure IoT Authentication Method () --->`. The default option is `Symmetric Key`:
+
+Parameter | Value
+---------|----------
+ `Azure IoT Device Symmetric Key` | _{Your Azure IoT Hub device symmetric key}_
+
+If you would like to use x509 certificates, select `X509 Certificates` and update the following values:
+
+Parameter | Value
+---------|----------
+ `Azure IoT Device Client Certificate` | _{Your Azure IoT Hub device certificate}_
+ `Azure IoT Device Client Certificate Private Key` | _{Your Azure IoT Hub device certificate private key}_
+
+Note that the certificate and private key must be a single line string with `\n` characters at the appropriate line breaks. For example:
+
+```txt
+# PEM Formatted (WRONG)
+-----BEGIN CERTIFICATE-----
+MIIBJDCBywIUfeHrebBVa2eZAbouBgACp9R3BncwCgYIKoZIzj0EAwIwETEPMA0G
+...
+vTfQahwsxN3xink9z1gtirrjQlqDAiEAyU+6TUJcG6d9JF+uJqsLFpsbbF3IzGAw
+yC+koNRC0MU=
+-----END CERTIFICATE-----
+
+# Single Line (CORRECT)

```

Comment:
Suggestion: show the correct way of doing it right at the top first, and then the comment "Note that the certi..." and finally the wrong example. Seems more intuitive like that.